nncf/quantization/quantize_model.py

```diff
@@ -226,22 +226,22 @@ def quantize_with_accuracy_control(
 
 
 @api(canonical_alias="nncf.compress_weights")
-def compress_weights(model: TModel, use_fake_quantize: bool = False) -> TModel:
+def compress_weights(model: TModel) -> TModel:
     """
     Compress model weights.
```

Comment:
It is not an API-breaking change, since the `compress_weights` functionality was introduced in this release.
See https://github.com/openvinotoolkit/nncf/pull/1941 for details.